src/applications/personalization/profile/tests/e2e/direct-deposit/v1-update-flow.cypress.spec.js

```diff
@@ -0,0 +1,98 @@
+import disableFTUXModals from '~/platform/user/tests/disableFTUXModals';
+import { PROFILE_PATHS } from '@@profile/constants';
+
+import mockUserInEVSS from '@@profile/tests/fixtures/users/user-36.json';
+import mockDD4CNPNotEnrolled from '@@profile/tests/fixtures/dd4cnp/dd4cnp-is-not-set-up.json';
+import mockDD4CNPEnrolled from '@@profile/tests/fixtures/dd4cnp/dd4cnp-is-set-up.json';
+import mockDD4EDUEnrolled from '@@profile/tests/fixtures/dd4edu/dd4edu-enrolled.json';
+
+// this entire test file can be removed when DD4EDU is on prod to 100% of users
+const dd4eduDisabled = {
+  data: {
+    type: 'feature_toggles',
+    features: [
+      {
+        name: 'ch33_dd_profile',
+        value: false,
```

Comment:
Making sure this feature flag is turned off so we got the old version of Direct Deposit